flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/FlinkPlannerImpl.scala

```diff
@@ -68,12 +68,16 @@ class FlinkPlannerImpl(
     }
   }
 
-  @throws(classOf[SqlParseException])
+  @throws(classOf[FlinkSqlParseException])
```

Comment:
Since **FlinkRelConversionException** is an unchecked exception, should we add it to **throws**?